velox/functions/prestosql/MultimapFromEntries.h

```diff
@@ -60,6 +54,17 @@ struct MultimapFromEntriesFunction {
       }
     }
   }
+
+ private:
+  // Use std::unordered_map to ensure deterministic order of keys in the
+  // result. Without ensuring deterministic order of keys, the results of
+  // expressions like map_keys(multimap_from_entries(...)) will be
+  // non-deterministic and trigger Fuzzer failures. F14Map is faster, but in
+  // debug builds it returns keys in non-deterministic order (on purpose).
+  std::unordered_map<
```

Comment:
Any reason for not using `folly::F14FastMap` ?

Reply:
The reason is explained in the comment right here :-)